webapp/components/Menu.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <nuxt-link to="/login" v-if="!loggedIn">Login</nuxt-link>
+    <button @click="logout" v-if="loggedIn">Logout</button>
+  </div>
+</template>
+
+<script>
+import {mapActions, mapGetters} from "vuex";
+
+export default {
+  name: "LoginForm",
+  computed: {
+    ...mapGetters('auth', ['loggedIn'])
+  },
+  methods: {
+    ...mapActions('auth', ['logout'])
+  }
+};
+</script>
```

Comment:
⭐ ⭐ For a menu component which shows a login or logout button and its software tests.